sdcm/mgmt/cli.py

```diff
@@ -851,8 +851,9 @@ def __init__(self, manager_node):
         ScyllaManagerBase.__init__(self, id="MANAGER", manager_node=manager_node)
         self._initial_wait(20)
         LOGGER.info("Initiating Scylla-Manager, version: {}".format(self.sctool.version))
-        list_supported_distros = [Distro.CENTOS7, Distro.DEBIAN8, Distro.DEBIAN9, Distro.DEBIAN10,
-                                  Distro.UBUNTU16, Distro.UBUNTU18, Distro.UBUNTU20]
+        list_supported_distros = [Distro.CENTOS7,
+                                  Distro.DEBIAN10, Distro.DEBIAN11,
+                                  Distro.UBUNTU18, Distro.UBUNTU20]
         self.default_user = "centos"
```

Comment:
Change to this file contradicts the statement made in the commit message because "list_supported_distros" is not only extended to have new distro support but also loses 3 other.